README.md

```diff
@@ -1,2 +1,462 @@
+from ebates_prod.temp.some_table
+```
+
+## Dont start column names with commas
+
+```
+-- Good
+select distinct
+    interest_area,
+    interest_score,
+    i.member_id,
+    order_merchant_id as store_id
+from ebates_prod.dw.order_transactions tx
+
+-- Bad
+select distinct
+    interest_area
+    , interest_score
+    , i.member_id
+    , order_merchant_id
+from ebates_prod.dw.order_transactions
+```
+
+## Join order & key order same
+
+Specify the order of a join with the FROM table first and JOIN table second. In the below example, `order_transactions` is joined first on `member_ftbs_by_store`, so the `on` condition should be in the same order, `tx.member_id = ftb.member_id` instead of `ftb.member_id = tx.member_id`
+
+```
+-- Good
+select distinct
+    interest_area,
+    interest_score,
+    i.member_id as user_id 
+    tx.order_merchant_id as store_id
+from ebates_prod.dw.order_transactions tx
+join ebates_prod.summary.member_ftbs_by_store ftb 
+    on tx.member_id = ftb.member_id
+
+-- Bad
+select
+    tx.order_merchant_id as store_id,
+    to_date(tx.click_date) as ds,
+    sum(tx.ot_amount) as sales_net,
+    round(ort_rebate_percentage * 100 * sales_net, 1) as commission_dummy
+from ebates_prod.dw.order_transactions tx
+join ebates_prod.summary.member_ftbs_by_store ftb 
+    on ftb.member_id = tx.member_id
+```
+
+## Indent should be four spaces
+
+Use four spaces instead of tabs for indentation. Configure editor to convert tabs to spaces
+
+```
+-- Good
+select distinct
+    interest_area,
+    interest_score,
+    i.member_id as user_id 
+    tx.order_merchant_id as store_id
+from ebates_prod.dw.order_transactions tx
+join ebates_prod.summary.member_ftbs_by_store ftb 
+    on ftb.member_id = tx.member_id
+
+-- Bad
+select distinct
+ interest_area,
+ interest_score,
+ i.member_id as user_id 
+ tx.order_merchant_id as store_id
+from ebates_prod.dw.order_transactions tx
+join ebates_prod.summary.member_ftbs_by_store ftb 
+on tx.member_id = ftb.member_id
+```
+
+## Use CTE's
+
+- Queries within the CTE's should be one level indented (4 spaces)
+- All the queries within CTE's should follow rest of the guidelines 
+- Commas seperating queries/CTE's should be at the end of each query/CTE instead of begining of next CTE, For example
+
+```
+-- Good
+with top10_interests as (
+
+    select distinct interest_area, interest_score, member_id 
+    from ff_last_year_member_interest_area
+
+    ),
+
+last_year_store_member_interest_area as (
+
+    select interest_area, rank as top10_rank 
+    ...
+    ...
+
+-- Bad
+with top10_interests as (
+    select distinct interest_area, interest_score, member_id 
+    from ff_last_year_member_interest_area
+    )
+
+, last_year_store_member_interest_area as (
+    select interest_area, rank as top10_rank 
+    ...
+    ...
+```
+
+- If you have to do nesting more than once or use multiple sub queries within a query, use CTEs instead. They are easy to read. Give meaningful names to them
```

Comment:
Might add one thing that we suggest nested queries should not greater than 2 levels deep.

Reply:
I meant the same by _If you have to do nesting more than once_